karim.html

```diff
@@ -234,6 +239,11 @@
 
     <label>Update all invoices <input type="submit" id="invoices" /></label>
     <template is="dom-bind" id="app">
+      <div>
+        <textarea>[[csvNames]]</textarea>
```

Comment:
Um. You can select the text then I thought? Lets merge this one now and we can update later. The committee needs it now :joy: